### PR TITLE
polymer and components initial structure added

### PR DESCRIPTION
**Pull request by romanfromrome**
plus login component

### CODE REVIEW

File: _client/client_app/angular_app/elements/iwish-app/iwish-app.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    <paper-button raised on-click="{{stuff}}">Next view in app</paper-button>
+
+  </template>
+  <script>
+
+
+    Polymer('iwish-app', {
+      selected: 1,
+
+         change : function () {
```
Comment (patyatka) [inline _client/client_app/angular_app/elements/iwish-app/iwish-app.html]: please fix identation

File: _client/client_app/angular_app/elements/iwish-app/iwish-app.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      var s = document.querySelector('select');
```
Comment (patyatka) [inline _client/client_app/angular_app/elements/iwish-app/iwish-app.html]: please rename this variable to something more descriptive. I can't understand what this element is...

File: _client/client_app/angular_app/elements/iwish-app/iwish-app.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      document.querySelector('core-animated-pages').transitions = document.querySelector('select').options[s.selectedIndex].value;
```
Comment (patyatka) [inline _client/client_app/angular_app/elements/iwish-app/iwish-app.html]: and here you write  document.querySelector('select') as was before is line 36

File: _client/client_app/angular_app/elements/iwish-app/iwish-app.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+<polymer-element name="iwish-app" attributes="">
+  <template>
+    <link rel="stylesheet" href="iwish-app.css">
+
+  <core-animated-pages selected="{{selected}}" transitions="cross-fade cross-fade-delayed scale-up slide-up slide-up-offscreen slide-down tile-cascade hero-transition">
+    <section>
+      <iwish-login-form layout vertical center center-justified slide-down response="{{out}}"></iwish-login-form>
+    </section>
+    <section>
+      <iwish-login-form layout vertical center center-justified slide-down></iwish-login-form>
```
Comment (patyatka) [inline _client/client_app/angular_app/elements/iwish-app/iwish-app.html]: 2 iwish-login-form elements. Where is iwish-forgot-form element?

File: _client/client_app/angular_app/elements/iwish-login-form/iwish-login/iwish-login.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+        <h1>
+          Hi there! Log in please:
+        </h1>
+      </div>
+      <div id="mainPanel" flex layout vertical slide-down>
+        <paper-input id="loginEmail" label="email" floatingLabel required error="input is required!"></paper-input>
+        <paper-input id="loginPass" label="password" floatingLabel required error="input is required!"></paper-input>
+        <div id="mainContainer" layout vertical?="{{!wide}}">
+          <paper-button raised id="login-button" auto-vertical disabled?="{{!$.loginPass.value.length || !$.loginEmail.value.length}}">login</paper-button>
+          <paper-button raised id="sign-up-button" auto-vertical on-click="{{animateCallback}}">sign up</paper-button>
```
Comment (patyatka) [inline _client/client_app/angular_app/elements/iwish-login-form/iwish-login/iwish-login.html]: is animateCallback inside paper-button element?

File: _client/shared_resources/stylesheets/main.scss [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+$main_bg_color: #F8F8F8;
+
+body {
+  background: $main_bg_color;
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  color: #333;
+}
+
+.hero-unit {
```
Comment (patyatka) [inline _client/shared_resources/stylesheets/main.scss]: do we use this file?

File: gulpfile.js [hunk 7]
```diff
@@ -148,18 +149,35 @@ gulp.task('styles', function () {
...
-    .pipe($.size({title: 'styles'}));
+    .pipe(gulp.dest(clientBuildPath))
+    .pipe($.size())
+});
+
+gulp.task('vulcanize', function(){
+  return gulp.src([
+    clientBuildPath + '/elements/elements.html'
+    ])
+        .pipe(vulcanize({
```
Comment (patyatka) [inline gulpfile.js]: indentation

File: _client/client_app/angular_app/elements/iwish-app/iwish-app.html (hunk 4 above)
Comment (romanfromrome) [inline _client/client_app/angular_app/elements/iwish-app/iwish-app.html]: in dom of <iwish-login-form>  element :)

**Pull request merged**